src/OpenSage.Game/Data/Ini/IniDataContext.cs

```diff
@@ -78,8 +93,18 @@ public sealed class IniDataContext
         public List<ObjectCreationList> ObjectCreationLists { get; } = new List<ObjectCreationList>();
         public OnlineChatColors OnlineChatColors { get; internal set; }
         public List<ParticleSystemDefinition> ParticleSystems { get; } = new List<ParticleSystemDefinition>();
+
+        [AddedIn(SageGame.Bfme)]
+        public Pathfinder Pathfinder { get; set; }
+
+        [AddedIn(SageGame.Bfme)]
+        public List<PlayerAIType> PlayerAITypes { get; } = new List<PlayerAIType>();
+
         public List<PlayerTemplate> PlayerTemplates { get; } = new List<PlayerTemplate>();
         public List<Rank> Ranks { get; } = new List<Rank>();
+
+        [AddedIn(SageGame.Bfme)]
+        public RegionCampain RegionCampain{ get; set; }
```

Comment:
Spotted a typo here, should be `RegionCampaign`.

Reply:
thats how its spelled inside the ini files, thought it would be easier to find in the code this way, but could rename it